Makefile

```diff
@@ -91,7 +78,7 @@ shellcheck:
 # the two.
 test-benchmark-compare:
 	@git fetch
-	./tests/compare_benchmarks.sh main
+	./tests/compare_benchmarks.sh master
```

Comment:
This likely should stay main branch

Reply:
yeah, didn't mean to change the makefile at all :-)